CMakeLists.txt

```diff
@@ -51,19 +51,7 @@ find_package(Git REQUIRED)
 option(USE_GIT_PROTOCOL
        "If behind a firewall turn this off to use HTTPS instead." OFF)
 
-option(AVX "Compile scan functions requiring AVX intrinsics" ON)
-option(AVX2 "Compile scan functions requiring AVX2 intrinsics" ON)
-
 # All options summary
-add_feature_info(AVX AVX "Complile scan functions requiring AVX intrinsics")
-add_feature_info(
-  AVX2 AVX2
-  "Compile scan functions requiring AVX2 intrinsics (enables AVX automatically)"
-)
-if(AVX2)
-  set(AVX ON)
-endif()
-
```

Comment:
This code path isn't needed. I added this a few months ago as a way to optionally turn on/off SIMD extensions, but there is a better way to do it.